cypress/tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     "target": "es6",
-    "lib": ["es6", "dom"],
+    "lib": ["esnext", "dom"],
```

Comment:
Bump ES up to `esnext` to match rest of webapp project, and allow access to ECMAScript stuff like `Object.entries`.